### DISCUSSION TITLE
overclock intel cpu

### DISCUSSION THREAD

**Discussion by nikitavedre** [opcm/pcm#533] [Q&A]
can i use this repository for overclocking?

**Comment by rdementi** [comment]
no, this is not the intended use case

**Comment by nikitavedre** [comment]
Do you know any repository for intel overclocking?

**Reply by ogbrugge** [reply]
Do you know this webpage?

https://www.intel.com/content/www/us/en/products/docs/processors/core/performance-maximizer.html

**Reply by ogbrugge** [reply]
Or this one?

https://www.intel.com/content/www/us/en/download/17881/intel-extreme-tuning-utility-intel-xtu.html

**Comment by nikitavedre** [comment]
yes ,I know these pages. Intel extreme tuning utility and intel performance maximizer is not working.

**Reply by ogbrugge** [reply]
Well you can always go into the BIOS and change the clock settings there, old school ;-)

Sorry but i have no other answer, PCM is not suited for overclocking, it is for measuring performance.

**Reply by nikitavedre** [reply]
ok thanks
